opm/autodiff/AutoDiffMatrix.hpp

```diff
@@ -558,6 +558,28 @@ namespace Opm
         }
 
 
+        /**
+         * Converts the AutoDiffMatrix to an Eigen SparseMatrix.This might be
+         * an expensive operation to perform for e.g., an identity matrix or a
+         * diagonal matrix.
+         */
```

Comment:
There is already a constructor that does this, is this needed separately?

(There is no move version of that constructor though.)